controllers/gatling_controller.go

```diff
@@ -418,7 +422,7 @@ func (r *GatlingReconciler) createVolumesForCR(ctx context.Context, gatling *gat
 		}
 	}
 	// Create PersistentVolume if defined to create in CR
-	if &gatling.Spec.PersistentVolumeSpec != nil && gatling.Spec.PersistentVolumeSpec.Name != "" {
+	if gatling.Spec.PersistentVolumeSpec.Name != "" {
```

Comment:
Could you remove the first condition?
I believe that if gatling.Spec.PersistentVolumeSpec is nil, trying to access gatling.Spec.PersistentVolumeSpec.Name would cause a panic.

Reply:
@gold-kou 
Thank you for the review.
I believe that gatling.Spec.PersistentVolumeSpec will never be nil.
This is because the PersistentVolumeSpec field variable in the GatlingSpec struct is not a pointer type.

https://github.com/st-tech/gatling-operator/blob/60bbeb037e3edd072ab0e660d05b2b43a2a239e9/api/v1alpha1/gatling_types.go#L59

Reply:
Got it !